streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImpl.java

```diff
@@ -109,89 +109,17 @@ public void setGlobalProcessorContext(final InternalProcessorContext globalProce
         this.globalProcessorContext = globalProcessorContext;
     }
 
-    @Override
```

Comment:
Same as above, we would leave the state directory locking etc at a single place. These changes here would break the global store update logic after this PR which I plan to resolve in the next PR -- I did not include the whole global state manager in this PR to reduce LOC.